Source/WebKit/UIProcess/BrowsingContextGroup.cpp

```diff
@@ -211,11 +213,11 @@ void BrowsingContextGroup::transitionPageToRemotePage(WebPageProxy& page, const
 
 void BrowsingContextGroup::transitionProvisionalPageToRemotePage(ProvisionalPageProxy& page, const Site& provisionalNavigationFailureSite)
 {
-    auto& set = m_remotePages.ensure(*page.page(), [] {
+    auto& set = m_remotePages.ensure(*page.protectedPage(), [] {
```

Comment:
I'm a bit unsure for this one, it seems it's only used as a key for the hash map...

Reply:
cc @rniwa for this one. It does seem unnecessary so maybe we can improve static analysis here. That said, this is not hot code so I don't mind the extra ref'ing here.